tests/Php73Test.php

```diff
@@ -3,8 +3,14 @@
 use Laravel\SerializableClosure\Exceptions\PhpVersionNotSupportedException;
 use Laravel\SerializableClosure\SerializableClosure;
 
-it('does not support PHP 7.3', function () {
+test('serializable closure does not support PHP 7.3', function () {
```

Comment:
You could prob. get rid of this and change to target php 8.1, as Laravel 10 targets that :)